source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak.md

```diff
@@ -54,10 +66,29 @@ following line:
 ]}
 ```
 
-Next, we need to expose the necessary Riak CS modules to Riak and to
-instruct Riak to use a the custom backend used by Riak CS. In the same
-`riak_kv` section from which we deleted the line shown above, we need
-to insert the following section:
+Next, we need to expose the necessary Riak CS modules to Riak and instruct Riak
+to use the custom backend provided by Riak CS. We need to use either the
+`advanced.config` or `app.config` file and insert the following options:
+
+```advancedconfig
+{riak_kv, [
+    %% Other configs
+    {add_paths, ["/usr/lib/riak-cs/lib/riak_cs-{{VERSION}}/ebin"]},
+    {storage_backend, riak_cs_kv_multi_backend},
+    {multi_backend_prefix_list, [{<<"0b:">>, be_blocks}]},
+    {multi_backend_default, be_default},
+    {multi_backend, [
+        {be_default, riak_kv_eleveldb_backend, [
+            {max_open_files, 50},
```

Comment:
Regarding what we described in release notes, I'd like this to be changed to `total_leveldb_mem_percent`. The value is ... say, 30?